lib/checkuninitvar.cpp

```diff
@@ -1112,6 +1112,10 @@ bool CheckUninitVar::isVariableUsage(const Token *vartok, bool pointer, Alloc al
     if (vartok->strAt(1) == "]")
         return true;
 
+    auto ast_parent = vartok->astParent();
```

Comment:
I would write `*` explicitly to show that it's a pointer. And we use camelCase for variable names.